org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -1,13 +1,58 @@
+from abc import ABC
```

Comment:
You could just ignore or delete this file and make changes to `VelocityTemplateFile.vm` because that's the one we are going to use at the end. Or at least make changes to both files.
After all, the only reason for me to use this file is that I don't want to keep seeing code that's not generated by our program during development; i.e., the base classes and the executor stuff.

Reply:
If you want people to ignore or delete a template file, then you probably shouldn't merge it into the repository in the first place. Yeah, I will make changes to both template files.

Reply:
I believe I have explained to you many times why I have this file. This file should not include the non-generated code because it's only used to test/develop. Now you are adding base class definitions to it, so it will not serve its original purpose anymore. That's why I hope you could just ignore it. Or maybe even delete it if you really wish to.

Reply:
I've made requested changes.